### ISSUE TITLE
Getting job done notifications for projects I don't lead

### ISSUE THREAD

**Issue by LimeeFox** [StealWonders/EpicJobs#4]
I am one of the leaders for 1 project, but there's another project which I do not lead yet still get job done notifications. I haven't tested whether I can complete them with fear that it might end up successful.

**Comment by StealWonders** [comment]
Do you suggest adding project deputies or rather a notification system to the plugin?

**Comment by LimeeFox** [comment]
Deputies, I guess. If I remember correctly, we briefly talked about having a possibility to have multiple project leaders for one project at once, but it was complicated or difficult.

**Comment by Tobberz** [comment]
I agree. AS well as assigning a Project Leader one should be able to have x number of deputies who also get perms for that project.
